package.json

```diff
@@ -72,7 +72,8 @@
   },
   "resolutions": {
     "yargs": "17.6.2",
-    "yoga-layout-prebuilt@^1.10.0": "patch:yoga-layout-prebuilt@npm%3A1.10.0#./.yarn/patches/yoga-layout-prebuilt-npm-1.10.0-855b15449f.patch"
+    "yoga-layout-prebuilt@^1.10.0": "patch:yoga-layout-prebuilt@npm%3A1.10.0#./.yarn/patches/yoga-layout-prebuilt-npm-1.10.0-855b15449f.patch",
+    "solidity-ast": "^0.4.45"
```

Comment:
this is another deps weirdness that is happening. We have to specify resolutions here (WITH a caret so we get latest) in order to have a newer version. If not, we are stuck with an old version that breaks coverage

Reply:
mhhhh That's really annoying :/